### PR TITLE
Feat update kria som commandline examples

### PR DESCRIPTION
**Pull request by LCY02**

### CODE REVIEW

File: examples/commandline_examples/k260_kria_som/input_usb_cam/usb_classification_visualization_rtsp.pbtxt [hunk 1]
```diff
@@ -41,8 +39,8 @@ node {
   name: "visualizer"
```
Comment (MatthewWiens101) [inline examples/commandline_examples/k260_kria_som/input_usb_cam/usb_classification_visualization_rtsp.pbtxt:39]: usb cannot output nv12 and I believe we do not have a color converter, so we cannot visualize the usb cam right now with video_sink unfortunately.

File: examples/commandline_examples/k260_kria_som/input_usb_cam/usb_detection-classification_visualization_rtsp.pbtxt [hunk 2]
```diff
@@ -69,8 +66,8 @@ node {
   name: "visualizer"
```
Comment (MatthewWiens101) [inline examples/commandline_examples/k260_kria_som/input_usb_cam/usb_detection-classification_visualization_rtsp.pbtxt:66]: same issue here about visualization of usb

**Review by MatthewWiens101 — CHANGES_REQUESTED** [review]
Other than the issues I noted with usb to visualizer, looks good! I think we can merge as soon as those are addressed.

File: examples/commandline_examples/k260_kria_som/input_usb_cam/usb_classification_visualization_rtsp.pbtxt (hunk 1 above)
Comment (LCY02) [reply examples/commandline_examples/k260_kria_som/input_usb_cam/usb_classification_visualization_rtsp.pbtxt]: fixed in [c6da292](https://github.com/auperatech/VMSS2.0/pull/39/commits/c6da29288e8da9d311e69e00e087a232ee96cc08)

**Pull request merged**